workers/unity/Assets/Generated/Source/improbable/gdk/tests/ExhaustiveBlittableSingularReaderWriter.cs

```diff
@@ -64,120 +64,360 @@ public ReaderWriterImpl(Entity entity, EntityManager entityManager, ILogDispatch
 
                 public event Action<BlittableBool> Field1Updated
                 {
-                    add => field1Delegates.Add(value);
-                    remove => field1Delegates.Remove(value);
+                    add
+                    {
+                        if (LogErrorIfDisposed())
```

Comment:
In future it would be really useful to order commits for this kind of thing. If there is a huge amount of stuff that is all the same then it's much easier to read if it can be isolated to a single commit, and the rest of the code in another (or several if there are multiple logical blocks)

Reply:
Yeah sorry about that. This was one of the situations where changes that others made in the meantime had to be pulled in and modified over and over.

Reply:
If the changes are simple then rebasing is probably fairly straightforward.